contributing/workflow.md

```diff
@@ -215,6 +214,33 @@ Label your PRs with the one of the following [labels](https://github.com/codeign
 And if your PRs have the breaking changes, label the following label:
 - **breaking change** ... PRs that may break existing functionalities
 
+## Updating Your Branch
+
+If you are asked for changes in the review, commit the fix in your branch and push it to GitHub again.
+
+If the `develop` branch progresses and conflicts arise that prevent merging, or if you are asked to *rebase*,
+do the following:
+
+Synchronize your repository:
+
+    git checkout develop
```

Comment:
Let's encourage the modern Guy commands:
```suggestion
    git switch develop
```

Reply:
Is anyone using a lower version that doesn't allow git switch anymore?

Reply:
I use `git switch my-branch` and find it more readable.

Reply:
It's almost three years old now, and 13 versions behind the current. There may still be people using older versions (I think Ubuntu 18.04 still ships lower?) but I would prefer for us to support the new nomenclature. If someone needs this tutorial they are likely on a fairly new install of Git.

Reply:
Okay, I updated. https://github.com/codeigniter4/CodeIgniter4/pull/6045/commits/dc3896d4d247117bc9643694227090b33dff3716